include/mbedtls/platform.h

```diff
@@ -116,12 +118,13 @@ extern void * (*mbedtls_calloc)( size_t n, size_t size );
 extern void (*mbedtls_free)( void *ptr );
 
 /**
- * \brief   Set your own memory implementation function pointers
+ * \brief   This function allows configuring custom memory-implementation 
```

Comment:
Grammar: “implementation” is a noun, and “memory implementation” is not a hyphenated compound, so why would there be a hyphen here?

Reply:
Because it was memory-implementation pointers - we hyphenate two or more words when they come before a noun they modify and form a single idea. This is called a compound adjective.